### PR TITLE
added new folder with 3 homework tasks

### PR DESCRIPTION
**Pull request by DMan113**
3d task didn't complete

### CODE REVIEW

**Comment by DMan113** [comment]
alreade done

File: DMan/HW3/1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name = input("Please, write your name:   ")
```
Comment (kolyasalubov) [inline DMan/HW3/1.py:1]: good job

File: DMan/HW3/2.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#  Якщо замовник бажає використовувати лише цілі значення, тоді ми маємо
+#  використовувати тип int, тобто:
+#  A = int(input("Please, enter the first number: ")) 
+
+
+
```
Comment (kolyasalubov) [inline DMan/HW3/2.py:6]: good job

File: DMan/HW3/3.1.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    48.Unless explicitly silenced.
+    49.In the face of ambiguity, refuse the temptation to guess.
+    50.There should be one-- and preferably only one --obvious way to do it.
+    51.Although that way may not be obvious at first unless you're Dutch.
+    52.Now is better than never.
+    53.Although never is often better than *right* now.
+    54.If the implementation is hard to explain, it's a bad idea.
+    55.If the implementation is easy to explain, it may be a good idea.
+    56.Namespaces are one honking great idea -- let's do more of those!
+    """
```
Comment (kolyasalubov) [inline DMan/HW3/3.1.py:57]: good job

File: DMan/HW3/3.2.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+user_number = input("Write please a four-digit number ")
```
Comment (kolyasalubov) [inline DMan/HW3/3.2.py:1]: good job

File: DMan/HW3/3.3.py [hunk 5]
```diff
@@ -0,0 +1,4 @@
+a = 10
```
Comment (kolyasalubov) [inline DMan/HW3/3.3.py:1]: good job

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**